src/bluetooth-fw/nimble/advert.c

```diff
@@ -16,17 +16,40 @@
 
 #include <bluetooth/bt_driver_advert.h>
 
-void bt_driver_advert_advertising_disable(void) {}
+#include "host/ble_gap.h"
+#include "system/logging.h"
+
+void bt_driver_advert_advertising_disable(void) {
+  int rc = ble_gap_adv_stop();
+  if (rc != 0) {
+    PBL_LOG_D(LOG_DOMAIN_BT, LOG_LEVEL_WARNING, "bt_driver_advert_advertising_disable err: %d", rc);
```

Comment:
```suggestion
    PBL_LOG_D(LOG_DOMAIN_BT, LOG_LEVEL_ERROR, "bt_driver_advert_advertising_disable err: %d", rc);
```

Reply:
Done (right now we hit this case whenever we stop the stack if a device was connected because the disconnection events aren't implemented).